common/src/main/java/net/dodogang/plume/client/PlumeClient.java

```diff
@@ -1,7 +1,20 @@
 package net.dodogang.plume.client;
 
+import net.dodogang.plume.Plume;
+import net.dodogang.plume.client.cosmetic.ClientCosmeticsManager;
+import net.dodogang.plume.cosmetic.CosmeticPlayerData;
+import net.dodogang.plume.cosmetic.Cosmetics;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.util.Identifier;
+
 public final class PlumeClient {
     public static void initialize() {
+        ClientCosmeticsManager.PLAYER_DATA.put(
+            MinecraftClient.getInstance().getSession().getProfile().getId(), CosmeticPlayerData.of(MinecraftClient.getInstance().getSession().getProfile().getId(), Cosmetics.MELON_MANGLER_HAT, Cosmetics.MELON_MANGLER_MASK, Cosmetics.PARTICLE)
+        );
```

Comment:
This is really confusing to read to me, so I would format it like this:

```java
        ClientCosmeticsManager.PLAYER_DATA.put(
                MinecraftClient.getInstance().getSession().getProfile().getId(), 
                CosmeticPlayerData.of(
                        MinecraftClient.getInstance().getSession().getProfile().getId(), 
                        Cosmetics.MELON_MANGLER_HAT, 
                        Cosmetics.MELON_MANGLER_MASK, 
                        Cosmetics.PARTICLE
                )
        );
```

Reply:
This won't be in the final version anyways as it's hardcoded cosmetics for the client user only :)